/**:
    ros__parameters:
        port: "/dev/ttyUSB1"
        hand_polarity: "L_"
        sensor_number: 5